src/vs/editor/common/config/fontInfo.ts

```diff
@@ -11,7 +11,7 @@ import { EditorZoom } from 'vs/editor/common/config/editorZoom';
  * Determined from empirical observations.
  * @internal
  */
-const GOLDEN_LINE_HEIGHT_RATIO = platform.isMacintosh ? 1.5 : 1.35;
+const GOLDEN_LINE_HEIGHT_RATIO = platform.isMacintosh ? 1.5 : 1.4;
```

Comment:
Will this change affect other editors?

Reply:
Very little, it will align better with the default values in Monaco.

Reply:
vs/workbench/browser/media/style.css:
.monaco-workbench {
	font-size: 13px;
	line-height: 1.4em;
	position: relative;
	z-index: 1;
	overflow: hidden;
}